draft-ietf-quic-qpack.md

```diff
@@ -871,21 +871,38 @@ the decoder.
 
 ### Blocked Eviction
 
-The encoder MUST NOT permit an entry to be evicted while a reference to that
-entry remains unacknowledged.  If a new header to be inserted into the dynamic
-table would cause the eviction of such an entry, the encoder MUST NOT emit the
-insert instruction until the reference has been processed by the decoder and
-acknowledged.
-
-The encoder can emit a literal representation for the new header in order to
-avoid encoding delays, and MAY insert the header into the table later if
-desired.
-
-To ensure that the blocked eviction case is rare, references to the oldest
-entries in the dynamic table SHOULD be avoided.  When one of the oldest entries
-in the table is still actively used for references, the encoder SHOULD emit an
-Duplicate representation instead (see {{duplicate}}).
+The encoder MUST NOT emit an instruction that evicts an entry while a reference
+to that entry remains unacknowledged.  When the encoder encounters a blocked
+eviction, it emits a literal representation for the new header, and MAY insert
+the header into the table later if desired.
+
+To ensure that blocked evictions are rare, the encoder SHOULD avoid referencing
```

Comment:
"To ensure that the encoder is not prevented from adding new entries, the encoder can avoid referencing entries that are near the point at which they would be evicted.  Rather than ..."